index.js

```diff
@@ -0,0 +1,19 @@
+const window = document.getElementById(page); // create a window variable so that we can eventually prepend the input and seconds to this
```

Comment:
Thanks for taking the time to do our short test.  I found a few errors in your submission.

`window` is a reserved word in JS and refers to the global object.  

Please test this file, fix up the errors, and resubmit.